ocfweb/docs/templates/docs/partials/account.md

```diff
@@ -0,0 +1,35 @@
+* GIMP (GNU Image Manipulation Program) is provided as an alternative to Adobe Photoshop, and Inkscape replaces Adobe Illustrator. For animation and 3D modeling, we provide access to Blender.
+* Rather than iTunes/QuickTime or Windows Media Player, you can access music or video files using VLC.
+* Pre-installed IDEs include Atom and Visual Studio Code.
+* The default PDF utility is Document Viewer, which can be used to view, print, and edit PDFs.
+* Steam is installed, if you would like to play video games.
+Most of these applications can be found on the desktop, and all applications can be accessed from the menu in the top-left corner of the screen, analogous to the Start menu in Windows. We also offer three computers with dedicated scanners, and one with a CD/DVD reader.
+
+## Persistent File Storage
+
+When you log out of one of our desktop computers, any files you saved to that computer will be erased. This is because most files saved to these computers are immediately printed, so there is no need to take up disk space holding onto them. However, some users may wish to have access to their files every time they log in to an OCF desktop. As part of your OCF account, you gain access to 15 GB of free file storage on OCF servers. This file storage is accessible by clicking on the "OCF File Storage" icon on any desktop computer or by SSHing into our account server (tsunami.ocf.berkeley.edu, and if you don't know what SSH means, read [our guide to the shell](LINK-GOES-HERE)).
+
+Files placed in the "OCF File Storage" folder are accessible from any desktop computer and are saved even after you log out, whereas files placed in the "Home" folder are local to that computer and are deleted upon log-out. The "OCF File Storage" folder should contain a folder called "public_html", which is used for web hosting – see below.
+
+## Free Printing
+
+By signing up for an account, you get 200 free sides of printing (200 pages single-sided, 100 pages double-sided). We print black-and-white only, in economode (meaning the ink may be a bit lighter than expected). You are limited to 20 sides of printing per day. We ask that you download or export any documents they wish to print as PDFs first, as these work best with our print server.
+
+## Web Hosting
+
+When you create an OCF account, you also gain acccess to web hosting space on our domain: ocf.berkeley.edu/\~yourusername. You can create your own website using HTML/CSS/JS or use a system like WordPress or Django. Alternatively, you can use your website as file storage. For more technical information on building your website, check out [our web hosting page](https://www.ocf.berkeley.edu/docs/services/web/).
+
+If you are a signatory of a student group and would like to request a \*.berkeley.edu website, you will need to create a group account and complete a few steps before making your request. Instructions can be found on [our vhost page](https://www.ocf.berkeley.edu/docs/services/vhost/).
```

Comment:
I wonder if it would be better to say "website under the berkeley.edu domain", in case some readers don't understand the asterisk...?